superchain/superchain_test.go

```diff
@@ -189,24 +190,23 @@ func testNetworkUpgradeTimestampOffset(l2GenesisTime uint64, blockTime uint64, u
 }
 
 func TestSuperchainConfigUnmarshaling(t *testing.T) {
-	rawYAML := `
-name: Mickey Mouse
-l1:
-  chain_id: 314
-  public_rpc: https://disney.com
-  explorer: https://disneyscan.io
-
-protocol_versions_addr: "0x252CbE9517F731C618961D890D534183822dcC8d"
-superchain_config_addr: "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"
-
-canyon_time: 1
-delta_time: 2
-ecotone_time: 3
-fjord_time:
+	rawTOML := `
+name = "Mickey Mouse"
+protocol_versions_addr = "0x252CbE9517F731C618961D890D534183822dcC8d"
+superchain_config_addr = "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"
+
+canyon_time = 1
+delta_time = 2
+ecotone_time = 3
+
+[l1]
+  chain_id = 314
+  public_rpc = "https://disney.com"
+  explorer = "https://disneyscan.io"
```

Comment:
toml's standard ordering puts all top level fields first, then nested fields follow. So this raw test data needed to be modified (along with the superchain.toml files) to adhere to that standard. Otherwise the fields are all missing when we unmarshal to a struct

Reply:
Not sure I follow: does this example not also have "all top level fields first, then nested fields follow"?

Reply:
This example was changed so that it has all top level fields first. Previously it looked like this:
```
name: Mickey Mouse
l1:
  chain_id: 314
  public_rpc: https://disney.com
  explorer: https://disneyscan.io

protocol_versions_addr: "0x252CbE9517F731C618961D890D534183822dcC8d"
superchain_config_addr: "0x02d91Cf852423640d93920BE0CAdceC0E7A00FA7"

canyon_time: 1
delta_time: 2
ecotone_time: 3
fjord_time:
```
My comment was just explaining the reason for the diff

Reply:
Gotcha! Makes sense, thanks.